bin/redshift/run_mvn_tests_on_aws.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Runs Redshift connector tests.
+#
+# Run setup_aws_redshift.sh as a prerequisite for creating the Redshift instance and viewing
+# required environment variables.
+#
+# Usage:
+# run_mvn_tests_on_aws.sh
+
+set -xeuo pipefail
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+cd "$ROOT_DIR"
+
+trap terminate INT TERM EXIT
+
+suite_exit_code=0
+
+MAVEN_OPTS="$MAVEN_INSTALL_OPTS" ./mvnw -s .github/settings.xml\
+    test \
+    -B -Dair.check.skip-all=true -Dmaven.javadoc.skip=true --fail-at-end \
+    -Dtest.redshift.jdbc.user="${REDSHIFT_USER}" \
+    -Dtest.redshift.jdbc.password="${REDSHIFT_PASSWORD}" \
+    -Dtest.redshift.jdbc.endpoint="${REDSHIFT_ENDPOINT}/" \
+    -Dtest.redshift.s3.tpch.tables.root="${REDSHIFT_S3_TPCH_TABLES_ROOT}" \
+    -Dtest.redshift.iam.role="${REDSHIFT_IAM_ROLES}" \
+    -pl :trino-redshift ||
+    suite_exit_code=1
+
+trap - INT TERM EXIT
+
+echo "$0: exiting with ${suite_exit_code}"
+exit "${suite_exit_code}"
```

Comment:
instead  of this, just invoke `./mvnw` with `exec`

Reply:
Would the command

```
trap - INT TERM EXIT
```

be still executed afterwards?